docassemble/ALWeaver/data/questions/template_validation.yml

```diff
@@ -585,37 +612,36 @@ subquestion: |
   
   All of the fields in your DOCX ${ interview.uploaded_templates.as_noun("file") } are listed
   in the table below. Confirm that the table looks right.
-  
-  A **bold** name means that this field is **reserved**.
-  Reserved fields are fields that will be handled with a question that
-  comes from the AL Question Library. For example:
-  
-  * Name fields
-  * Address fields
-  * Basic contact information and court information
-  
-  If you expect a field to be bold but it isn't, check the field's spelling.
-
+        
   The **type** of each field is just a guess, based on the field's name.
   You can override the guess later.
 
-  Name (bold if {reserved}) | Type
+  Name | Type
   --------------------------|------
   % for field in validation_fields.builtins():
-  ${ bold(field.variable) } :question: | ${ field.field_type_guess } |
+  % if is_reserved_docx_label( field.variable, reserved_pluralizers_map=dict(enumerate(people_list)) ):
+  ${ bold(field.variable) } :person: | ${ field.field_type_guess } |
+  % else:
+  ${ bold(field.variable) } | ${ field.field_type_guess } |
+  % endif
+  % endfor
+  % for field in validation_fields.reserved():
+  `${ field.variable }` | |
   % endfor
   % for field in validation_fields.custom():
   ${ bold(field.variable) if field.reserved else field.variable } |  ${ field.field_type_guess } | 
   % endfor
   % for field in validation_fields.signatures():
   ${ bold(field.variable) if field.reserved else field.variable } | signature  |
   % endfor
-  
-  % if len(validation_people_list) > 0:
-  :question: Indicates that we think this field could be handled by 
-  questions in our question library, so you don't have to write your own 
-  questions for it! You can choose whether to use our default questions later.
-  % endif
+
+  Key:
+    
+  Symbol   | Meaning
+  ---------|---------
+  **bold** | Handled by a question in our question library
+  :person: | Can be optionally handled as an object representing a person
+  `red`    | A word with a reserved meaning in Docassemble, and will not have a question associated with it
```

Comment:
Useful, but I don't see it used?

Reply:
You need a file with person candidates. Like: person[0].name.full()

Reply:
I misunderstood, I thought it meant the symbol would be the color red, not that it was talking about the code formatting. That's fine.